pkg/proxy/proxy_test.go

```diff
@@ -1168,7 +1245,7 @@ func (s *TestProxySuite) token(userID uuid.UUID, extraClaims ...authsupport.Extr
 		Username: "username-" + userID.String(),
 	}
 
-	extra := append(extraClaims, authsupport.WithEmailClaim("someemail@comp.com"))
+	extra := append([]authsupport.ExtraClaim{authsupport.WithEmailClaim("someemail@comp.com")}, extraClaims...)
```

Comment:
for context: this change allows us to override the default email set by this auxiliary function. As a result if you don't define an email, you'll have `someemail@comp.com`. Otherwise you'll have the one you are explicitly setting

Reply:
Worth moving this comment to the code.

Reply:
done in d81542b35ef008098f4c3b09b4fbf1b9aeab453e